CHANGES.md

```diff
@@ -39,8 +39,10 @@ Release Notes.
 * Add `normal` to V9 service meta query.
 * Support `scope=ALL` catalog for metrics.
 * Bump up H2 to 2.1.210 to fix CVE-2022-23221.
-* E2e: Add `normal` field to Service.
+* E2E: Add `normal` field to Service.
 * Add FreeSql component ID(3017) of dotnet agent.
+* E2E: verify OAP cluster model data aggregation.
+* Fix `SelfRemoteClient` self observing metrics.
```

Comment:
Miss 2 change logs
1. Expose those 2 configurations for several coordinator in the configuration
2. Doc update about (1).

Reply:
done